python_modules/dagster/dagster/_cli/asset.py

```diff
@@ -78,9 +78,11 @@ def execute_materialize_command(instance: DagsterInstance, kwargs: Mapping[str,
     else:
         tags = {}
 
-    execute_job(
+    result = execute_job(
         job=reconstructable_job, asset_selection=list(asset_keys), instance=instance, tags=tags
     )
+    if not result.success:
+        raise click.ClickException("Materialization failed.")
```

Comment:
is there a way to know what asset failed and report it here?

Reply:
likely, though the output of running the CLI contains the events logged out which has the full context of failure already 

example here: https://gist.github.com/alangenfeld/04639f6ebaa7bcbc962bd6028731ebbf

Reply:
ok! does't seem worth it then